devfiles/quarkus-commandmode/devfile.yaml

```diff
@@ -0,0 +1,100 @@
+---
```

Comment:
should the directory be `quarkus-command-mode` (two dashes) ?

Reply:
fixed